### PR TITLE
Feature/add sepolia amoy network

### PR DESCRIPTION
**Pull request by abbesBenayache**

### CODE REVIEW

File: tests/scripts/prepare-bellecour-fork-for-tests.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const PROD_WORKERPOOL_OWNER_WALLET =
   '0x1Ff6AfF580e8Ca738F76485E0914C2aCaDa7B462';
 const DEBUG_WORKERPOOL = '0xdb214a4a444d176e22030be1ed89da1b029320f2'; // 'debug-v8-bellecour.main.pools.iexec.eth';
 const PROD_WORKERPOOL = '0x0e7bc972c99187c191a17f3cae4a2711a4188c3f'; // 'prod-v8-bellecour.main.pools.iexec.eth';
-const ORACLE_DAPP_ADDRESS = '0xd11f5d70f8817add3d9e15d316911e6a4d699f79'; // oracle-factory.apps.iexec.eth
```
Comment (PierreJeanjacquot) [inline tests/scripts/prepare-bellecour-fork-for-tests.js:21]: using hardcoded addresses is not ideal as ENS will change and test will break.
here is how we fixed it in another project
https://github.com/iExecBlockchainComputing/web3mail-sdk/blob/00dcc4976d6152020e33d3d4128a35e82c25e322/tests/scripts/prepare-bellecour-fork-for-tests.js#L22-L42

File: src/config/config.ts [hunk 2]
```diff
@@ -34,23 +34,33 @@ const readerConfMap: Record<number, { ORACLE_CONTRACT_ADDRESS: AddressOrENS }> =
...
+  sepolia: 'sepolia',
 };
 
 const DEFAULT_IPFS_GATEWAY: string = 'https://ipfs-gateway.v8-bellecour.iex.ec';
 
 const DEFAULT_IPFS_UPLOAD_URL: string =
   '/dns4/ipfs-upload.v8-bellecour.iex.ec/https';
 
-const SUPPORTED_TARGET_BLOCKCHAINS: number[] = [1, 137];
+const SUPPORTED_TARGET_BLOCKCHAINS: number[] = [1, 137, 11155111, 80002];
```
Comment (PierreJeanjacquot) [inline src/config/config.ts:63]: use numerical order?

File: tests/e2e/IExecOracleFactory/updateOracle.e2e.test.ts [hunk 3]
```diff
@@ -479,7 +479,7 @@ describe('oracleFactory.updateOracle()', () => {
                 method: 'GET',
                 url: 'https://foo.io',
               },
-              { useVoucher: false }
+              { targetBlockchains: [134, 80002, 11155111], useVoucher: false }
```
Comment (PierreJeanjacquot) [inline tests/e2e/IExecOracleFactory/updateOracle.e2e.test.ts:482]: wrong chainId
```suggestion
              { targetBlockchains: [137, 80002, 11155111], useVoucher: false }
```

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]

File: tests/scripts/prepare-bellecour-fork-for-tests.js (hunk 1 above)
Comment (abbesBenayache) [reply tests/scripts/prepare-bellecour-fork-for-tests.js:21]: commit : e6967f0e19fee872ca41e5c11397b5338c6d9b79

File: src/config/config.ts (hunk 2 above)
Comment (abbesBenayache) [reply src/config/config.ts:63]: commit : 6cfeb43fe0cedb449b6a44ad430bd266f2fc5b5e

File: tests/e2e/IExecOracleFactory/updateOracle.e2e.test.ts (hunk 3 above)
Comment (abbesBenayache) [reply tests/e2e/IExecOracleFactory/updateOracle.e2e.test.ts:482]: commit : fb7110d671ff9fffca12b9b1a7e36c71a0aab499

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Pull request merged**